boards/FSM/Inc/App/configs/App_SignalCallbackDurations.h

```diff
@@ -8,3 +8,5 @@
 #define APPS_EXIT_HIGH_MS 1000
 #define APPS_AND_BRAKE_ENTRY_HIGH_MS 1
 #define APPS_AND_BRAKE_EXIT_HIGH_MS 1
+#define FLOW_METER_ENTRY_HIGH_MS 1000
+#define FLOW_METER_EXIT_HIGH_MS 1000
```

Comment:
these defines should be unsigned (i.e. 1000U) so that you're not comparing unsigned and signed ints (assuming that the time variables you will compare/store these with are unsigned)

also, every define you make should have brackets around every number and variable, i.e. `#define yar (1000U)` - https://stackoverflow.com/a/9081708

could be worth doing a dedicated PR for that instead of doing it here

Reply:
![image](https://user-images.githubusercontent.com/32230980/102005700-be16e380-3ccf-11eb-8a10-d597d8d9768c.png)

Reply:
This should be addressed now, king

Reply:
in all .h files? or just this one?

Reply:
Just this one